src/main/java/de/sldk/mc/metrics/WorldSize.java

```diff
@@ -0,0 +1,43 @@
+package de.sldk.mc.metrics;
+
+import io.prometheus.client.Gauge;
+import java.io.File;
+import org.bukkit.World;
+import org.bukkit.plugin.Plugin;
+
+public class WorldSize extends WorldMetric {
+
+    private static final Gauge WORLD_SIZE = Gauge.build()
+            .name(prefix("world_size"))
+            .help("World size in bytes")
+            .labelNames("world")
+            .create();
+
+    public WorldSize(Plugin plugin) {
+        super(plugin, WORLD_SIZE);
+    }
+
+    @Override
+    protected void clear() {
+        WORLD_SIZE.clear();
+    }
+
+    @Override
+    public void collect(World world) {
+        File worldFolder = world.getWorldFolder();
+        long size = getFolderSize(worldFolder);
+        WORLD_SIZE.labels(world.getName()).set(size);
+    }
+
+    private long getFolderSize(File folder) {
+        long size = 0;
+        for (File file : folder.listFiles()) {
```

Comment:
My IDE tells me that `folder.listFiles()` might produce a NullPointerException. Can you add a null check or consider using [this](https://stackoverflow.com/a/19877372/1510659) which seems more robust (but also more verbose :sweat_smile:)?

Reply:
I like that implementation, seem faster